model/security-rule/registry.yaml

```diff
@@ -0,0 +1,60 @@
+groups:
+  - id: registry.security_rule
+    display_name: Security Rule
+    type: attribute_group
+    brief: >
+      Describes security rule attributes. Rule fields are used to capture the specifics of any observer or agent rules
+      that generate alerts or other notable events.
+    attributes:
+      - id: security_rule.category
+        type: string
+        stability: experimental
+        brief: >
+          A categorization value keyword used by the entity using the rule for detection of this event
+        examples: ['Attempted Information Leak']
+      - id: security_rule.description
+        type: string
+        stability: experimental
+        brief: >
+          The description of the rule generating the event.
+        examples: ['Block requests to public DNS over HTTPS / TLS protocols']
+      - id: security_rule.license
+        type: string
+        stability: experimental
+        brief: >
+          Name of the license under which the rule used to generate this event is made available.
+        examples: ['Apache 2.0']
+      - id: security_rule.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the rule or signature generating the event.
+        examples: ['BLOCK_DNS_over_TLS']
+      - id: security_rule.reference
+        type: string
+        stability: experimental
+        brief: >
+          Reference URL to additional information about the rule used to generate this event.
+        note: >
+          The URL can point to the vendor’s documentation about the rule.
+          If that’s not available, it can also be a link to a more general page describing this type of alert.
+        examples: ['https://en.wikipedia.org/wiki/DNS_over_TLS']
+      - id: security_rule.ruleset.name
```

Comment:
@lmolkova I have updated this field as discussed and resolved previous conversation as it was pointing to the old file. Thank you for your input